main/template/default/social/avatar_block.tpl

```diff
@@ -18,7 +18,7 @@
             </div>
         {% elseif show_user %}
             <a href="{{ user_image.big }}" class="expand-image">
-                <img class="img-responsive img-circle" src="{{ user_image.big }}">
+                <img class="img-responsive img-circle" src="{{ user_image.big }}" alt="Avatar">
```

Comment:
I'm not sure the term "Avatar" is correct in all languages. In particular in non-roman letters languages like Chinese, Greek, Russian, Arabic, etc. Please use a language variable for that. I have just added a term "$Avatar" to the language files, so please use {{ 'Avatar' | get_lang }} instead of the current form.